Add checkbox to help contribute (#10)
holoviz/.github: .github/ISSUE_TEMPLATE/bug_report.md

#### Description of expected behavior and the observed behavior

> - [ ] I'd like to create a pull request to address this

Reviewer: ```suggestion
- [ ] I may be interested in making a pull request to address this
```

Author: I was also thinking of adding another checkbox: `- [ ] I need step-by-step guidance`

Reviewer 2: I would maybe put this at the end of the file.
